dsl/pace/dsl/stencil.py

```diff
@@ -189,12 +191,46 @@ def __init__(
         stencil_function = gtscript.stencil
         stencil_kwargs = {**self.stencil_config.stencil_kwargs}
 
+        if "dace" in self.stencil_config.backend:
```

Comment:
Does this imply revision to the delnflux or remapping code?

Reply:
Good question. This is a global config setting for dace, so yes it should be propagated everywhere. The part I'm not sure is whether we want to do it here, or somewhere else that gets triggered whenever dace backend is triggered.